gensim/models/wrappers/fasttext.py

```diff
@@ -233,11 +233,12 @@ def load_fasttext_format(cls, model_file, encoding='utf8'):
 
         `model_file` is the path to the FastText output files.
         FastText outputs two training files - `/path/to/train.vec` and `/path/to/train.bin`
-        Expected value for this example: `/path/to/train`
+        Expected value for this example: `/path/to/train`. However, you only need .bin
+        file to load the entire model.
 
         """
         model = cls()
-        model.wv = cls.load_word2vec_format('%s.vec' % model_file, encoding=encoding)
+        model.file_name = model_file
         model.load_binary_data('%s.bin' % model_file, encoding=encoding)
```

Comment:
I wonder if it would be a good idea to allow taking the whole model filename (including the .bin) as valid input - with the latest changes, we're loading from the bin file only, so it makes intuitive sense for the entire filename to be valid input. 
The only reason IMO we're still allowing the filename without extension as valid input is backward compatibility.